presto-generation-processor/src/main/resources/ConnectorTest.vt

```diff
@@ -0,0 +1,20 @@
+package ${package};
+
+import static ${fqConnectorClass}.getSupplier;
```

Comment:
Maybe add a check in the Processor that classes annotated @TestConnector contain a .getSupplier method for better error generation?